src/mame/video/k057714.h

```diff
@@ -48,6 +61,8 @@ class k057714_device : public device_t
 
 	void draw_frame(int frame, bitmap_ind16 &bitmap, const rectangle &cliprect, bool inverse_trans);
 
+	required_device<screen_device> m_screen;
```

Comment:
Would it be better to use the `device_video_interface` mix-in to pick up the screen?

Reply:
Yeah, that's probably what I should've been using, looking at some other similar devices now. Thanks.